bigtable/README.md

```diff
@@ -7,9 +7,25 @@
 [quickstart-link]: http://GoogleCloudPlatform.github.io/google-cloud-cpp/
 
 This directory contains the implementation of the Google Cloud Bigtable C++
-client. This is a work-in-progress and is not yet suitable for production use.
+client.
+
+## Status
+
+This library support Google Cloud Bigtable at the
+[Alpha](../README.md#versioning) quality level.
 
 ## Documentation
 
 Full documentation, including a [quick start guide][quickstart-link] 
 is available [online][doxygen-link].
+
+## Release Notes
```

Comment:
Can you please add:

1. a version number, and
2. a date of release

so that this makes sense to readers in the future?

Reply:
Yup.  I sent a small PR to fix this.

I think I should prepare a doc on how to create releases too, it is somewhat tricky.